<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetValue</class>
 <widget class="QDialog" name="DialogSetValue">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>172</width>
    <height>90</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>设值</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTextEdit" name="textEditCurrentSV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>当前值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="placeholderText">
    <string>设置值</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditUnit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>50</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>℃</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
